gen2-deeplabv3_depth/main.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python3
+
+from pathlib import Path
+import cv2
+import depthai as dai
+import numpy as np
+import argparse
+import time
+import sys
+
+'''
+Blob taken from the great PINTO zoo
+
+git clone git@github.com:PINTO0309/PINTO_model_zoo.git
+cd PINTO_model_zoo/026_mobile-deeplabv3-plus/01_float32/
+./download.sh
+source /opt/intel/openvino/bin/setupvars.sh
+python3 /opt/intel/openvino/deployment_tools/model_optimizer/mo_tf.py   --input_model deeplab_v3_plus_mnv2_decoder_256.pb   --model_name deeplab_v3_plus_mnv2_decoder_256   --input_shape [1,256,256,3]   --data_type FP16   --output_dir openvino/256x256/FP16 --mean_values [127.5,127.5,127.5] --scale_values [127.5,127.5,127.5]
+/opt/intel/openvino/deployment_tools/inference_engine/lib/intel64/myriad_compile -ip U8 -VPU_NUMBER_OF_SHAVES 6 -VPU_NUMBER_OF_CMX_SLICES 6 -m openvino/256x256/FP16/deeplab_v3_plus_mnv2_decoder_256.xml -o deeplabv3p_person_6_shaves.blob
+'''
+
+parser = argparse.ArgumentParser()
+parser.add_argument("-nn", "--nn_model", help="select camera input source for inference", default='models/deeplab_v3_plus_mvn2_decoder_256_openvino_2021.2_6shave.blob', type=str)
+args = parser.parse_args()
+
+nn_path = args.nn_model
+
+
+nn_shape = 256
```

Comment:
let's make an argument `nn_shape` and make it default to `256` - checking for string in path can cause some unclear edge-cases (e.g. when user will provide an absolute path and have `513` in their username)